File: sentimental-cash/cash.py
from cs50 import get_float

quarters = 0.25
dimes = 0.10
nickels = 0.05
pennies = 0.01

while True:
    change = get_float("Change: ")
    if change >= 0:
        break

    print("Change cannot be negative.")

change_owed_cents = round(change * 100)
num_coins = 0

while change_owed_cents > 0:

    if change_owed_cents >= 25:
        change_owed_cents -=25
        num_coins += 1
    elif change_owed_cents >= 10:
        change_owed_cents -=10
        num_coins += 1
    elif change_owed_cents >= 5:
        change_owed_cents -= 5
        num_coins += 1
    else:
        change_owed_cents -= 1
        num_coins += 1



print(num_coins)

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// ask for change owed, in cents
    int cents;
    do
    {
        cents = get_int("Change owed, in cents: ");
    }
    while (cents < 0);

//  quarters
    int quarters = cents / 25;
    cents = cents % 25;

//  dimes
    int dimes = cents / 10;
    cents = cents % 10;

//  nickels
    int nickels = cents / 5;
    cents = cents % 5;

//  pennies
    int pennies = cents;

//  The result
    printf("Quarters: %d\n", quarters);
    printf("Dimes: %d\n", dimes);
    printf("Nickels: %d\n", nickels);
    printf("Pennies: %d\n", pennies);
}


int = int_quarters = 0.25
int = int_dimes = 0.10
int = int_nickels = 0.05
int = int_pennies = 0.01
